manual/index.rst

```diff
@@ -1,7 +1,7 @@
 WorldEngine - the best open-source world generator
```

Comment:
Well, the headline has to be shorter than "is intended to be the best open-source world generator".
I also searched many, many, many, many times for open-source world generator and I could not find any that is comparable to WorldEngine. If I can find one I would change the headline :D